solcjs

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var solc = require('./index.js');
+// FIXME: remove annoying exception catcher of Emscripten
+//        see https://github.com/chriseth/browser-solidity/issues/167
+process.removeAllListeners('uncaughtException');
```

Comment:
I think @chriseth only meant the `removeAllListeners` part. That must only be called under node.js. For some reason it doesn't seem to work reliably. It is annoying bug discussed in https://github.com/chriseth/browser-solidity/issues/167, any help welcome solving it.